### PR TITLE
Website header #1

### LINKED ISSUE #1: 🎨 Website header
#  Create the website header
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a header I can use to navigate the website.

### Description

Your task is to add the website header as described below.

The content of this header will include:

```
1. "Elewa" logo on the left
2. A transparent background 
3. Black text in correct font 
4. Contact button on the right
```

The header looks as follows:

image

 **_TODO_** Logo Url -> https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690297/elewa-group-website/Icons/SVG/_Logo/Logo_White_arvoyx.svg

The header needs to be absolute and of higher z-index so that it appears on top of the page banner.

Full design reference: https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/

The container adapts naturally to the following screen widths:

Device | Screen width | Container width
-- | -- | --
Large Screens (Desktops and Laptops) | 1440px | 1200px
Tablets | 768px | 728px
Mobiles | 360px | 320px

image

### Technical Analysis

- Create an angular package in libs/elements/layout/header. Name the module AppHeaderModule
- Create a header component inside a components folder of the package above.
- Export the header in the package 
- Use labels to make the header available in two languages

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The header adapts naturally to different screen widths
3. The header can change language

### PR DESCRIPTION
**Pull request by KaranjaK**
# Description

The changes effected are:
1. Creation o a header.
2. Addition of the Menu List and close icons in to power responsiveness of the header once loaded on Tablet and phone screens. 
3. Added ngx translate dependency to power translation 

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes #1  - 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
image
image

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/elements/layout/header/src/lib/header.routing.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export const ELEWA_HOME_ROUTES: Route[] = [
+
+  { path: '', component: HeaderComponent },
+
+];
+
+@NgModule({
+  imports: [RouterModule.forChild(ELEWA_HOME_ROUTES)],
+  exports: [RouterModule]
+})
```
Comment (LemmyMwaura) [inline libs/elements/layout/header/src/lib/header.routing.ts:15]: Header does not need to have routing set

File: libs/features/pages/home/src/lib/home.routing.ts [hunk 2]
```diff
@@ -1,11 +1,8 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Route } from '@angular/router';
 
-import { HomePageComponent } from './main/home/home-page.component';
-
 export const ELEWA_HOME_ROUTES: Route[] = [
 
-  { path: '', component: HomePageComponent },
 
 ];
```
Comment (LemmyMwaura) [inline libs/features/pages/home/src/lib/home.routing.ts:7]: Should still be here. The home page needs the route to be set

File: package.json [hunk 3]
```diff
@@ -26,7 +26,10 @@
     "@angular/router": "16.1.7",
     "@ngneat/transloco": "^5.0.2",
     "@nguniversal/express-engine": "^16.1.1",
+    "@ngx-translate/core": "^15.0.0",
+    "@ngx-translate/http-loader": "^8.0.0",
     "@nrwl/angular": "16.6.0",
+    "bootstrap-icons": "^1.10.5",
```
Comment (LemmyMwaura) [inline package.json:32]: We use font-awesome icons. I'll add a CDN for this that you can get by syncing the latest changes

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

**Comment by LemmyMwaura** [comment]
Kindly resolve the issues above so that we merge this

**Comment by KaranjaK** [comment]
Well noted. Kindly allow me to do it tomorrow morning before 6am

**Comment by LemmyMwaura** [comment]
We are closing this as it's been resolved in #26. Suppose you need feedback reply below.

**Pull request closed without merging**